packages/create-sitecore-jss/src/templates/nextjs-xmcloud/src/Bootstrap.tsx

```diff
@@ -1,21 +1,34 @@
+import { useEffect, useState } from 'react';
 import { SitecorePageProps } from 'lib/page-props';
-import { context } from 'src/lib/context';
+import { CloudSDK } from '@sitecore-cloudsdk/core/browser';
+import '@sitecore-cloudsdk/events/browser';
 import config from 'temp/config';
+import { LayoutServicePageState } from '@sitecore-jss/sitecore-jss-nextjs';
 
 /**
  * The Bootstrap component is the entry point for performing any initialization logic
  * that needs to happen early in the application's lifecycle.
  */
 const Bootstrap = (props: SitecorePageProps): JSX.Element | null => {
-  /**
-   * Initializes the application Context and associated Software Development Kits (SDKs).
-   * This function is the entry point for setting up the application's context and any SDKs that are required for its proper functioning.
-   * It prepares the resources needed to interact with various services and features within the application.
-   */
-  context.init({
-    siteName: props.site?.name || config.sitecoreSiteName,
-    pageState: props.layoutData?.sitecore?.context?.pageState,
```

Comment:
Previously we were passing "pageState" to Components, so Forms component was able to have check for "pageState" to identify if "Form" event should be sent in editing/preview mode. Is it changed on "Forms" component's side now? How they will identify if mode is "editing"?

Reply:
The same question is related to "siteName", currently you just reference "config.sitecoreSiteName", however we need to resolve "siteName" dynamically based on "siteResolver", "multisiteMiddleware"

Reply:
Basically, we still need to be able to update BYOC context value, since some values are "runtime"